static/app/components/featureBadge.tsx

```diff
@@ -29,18 +32,37 @@ const labels = {
   new: t('new'),
 };
 
-function BaseFeatureBadge({type, variant = 'badge', title, noTooltip, ...p}: Props) {
+function BaseFeatureBadge({
+  type,
+  variant = 'badge',
+  title,
+  noTooltip,
+  expiresAt,
+  ...props
+}: Props) {
   const theme = useTheme();
+  if (expiresAt && expiresAt.valueOf() < Date.now()) {
+    // Only get 1% of events as we don't need many to know that a badge needs to be cleaned up.
+    if (Math.random() < 0.001) {
```

Comment:
@markstory shouldn't this be < 0.01? if you want 1% of events or to be more accurate <= 0.01 (I think you'll be collecting < 0.1% now)

I wonder if we should just report all of them and just let the backend handle the volume problem, or will that create a lot of noise in our dashboard?

Reply:
Whoops, you're right I have too many 0's. 

> I wonder if we should just report all of them and just let the backend handle the volume problem, or will that create a lot of noise in our dashboard?

I don't think we need all the volume from this though. If we get even 1 event we will have enough signal on where it is to remove it. Sending all the events would just heat the earth.